compose/service.py

```diff
@@ -667,7 +667,10 @@ def _get_container_host_config(self, override_options, one_off=False):
 
         devices = options.get('devices', None)
 
+        version = os.environ.get('COMPOSE_API_VERSION', '1.19')
+
         return create_host_config(
+            version=version,
```

Comment:
It's too bad `docker.Client()._version` is "private". It would have been nice to just use `self.client._version` instead of reading it from the environment again (which maybe is still an option)?

If we have to read it again, we should probably move the `COMPOSE_API_VERSION` string to `const.py`, and create a `DEFAULT_API_VERSION` in `const.py` and use it instead of hardcoding `1.19` in both places.